### ISSUE TITLE
Let's Encrypt certificate not accepted in App on Android 5.0.1

### ISSUE THREAD

**Issue by rrooggiieerr** [home-assistant/android#3145]
**Home Assistant Android version:**
2022.11.0-full

**Android version:**
5.0.1 (Yes, I know it's old)

**Phone model:**
Lenovo Tab3 7 Essential
lenovo tb3-710f

**Home Assistant version:**
2022-12-01

**Last working Home Assistant release (if known):**

**Description of problem:**
Since I'm using HTTPS for my connection to Home Assistant, instead to plain HTTP, the companion app on my old Lenovo tablet is no longer able to connect. Home Assistant is working trough the Chrome Browser on the same tablet using HTTPS though.

**Traceback (if applicable, to get the logs you may refer to: https://companion.home-assistant.io/docs/troubleshooting/faqs/#android-crash-logs):**

```

```

**Screenshot of problem:**
![Screenshot_2022-12-08-17-44-10](https://user-images.githubusercontent.com/18603534/206512806-86e1d024-0427-4c6c-a497-5cd05f237e2b.png)

**Additional information:**
According to Let's Encrypt their certificates should work on old Android devices till September 2024:
https://letsencrypt.org/2020/12/21/extending-android-compatibility.html
https://community.letsencrypt.org/t/production-chain-changes/150739

The fact that I can connect to my HA instance using the Chrome browser using HTTPS on the same tablet shows that the certificate is indeed working.

**Comment by dshokouhi** [comment]
Sorry but we are not able to solve SSL issues those need to be solved by the user. In this case it seems to be a case where the android device does not know the CA used. The android app cannot be compared to chrome here because the app makes use of API calls and android apps are bound to the known CA on the device where browsers are not. Every android device is different here and some can be pickier than others.

Please refer to some of the tips here on how to resolve the issue.

https://developer.android.com/training/articles/security-ssl#CommonProblems

We have seen some users get successful at either generating a new certificate or using a different provider which will have a different CA.

**Comment by marazmarci** [comment]
I didn't research this, but maybe you could add Let's Encrypt's CA to the device's trusted CAs manually (?)

**Comment by rrooggiieerr** [comment]
@marazmarci , actually the Let's Encrypt CA is in the trusted CAs. In the meantime I found out that the Let's Encrypt add-on uses a different preferred chain than default. I created an issue for that add-on and also a pull request which should fix it.

To answer your question further, yes, it's possible to add a CA to the device, but then you always have to login on the device on reboot and unlock, so that's not really a workable solution.
